Source/Core/Common/msvc_sd6_compat.h

```diff
@@ -0,0 +1,226 @@
+// Copyright 2013 Dolphin Emulator Project
```

Comment:
What's the purpose of these dates? Legally the number is meaningless, but maybe there's some other reason people want to know the date a file was started.